client/bindings/nodejs/Cargo.toml

```diff
@@ -1,9 +1,5 @@
-[workspace]
-
 [package]
-# NOTE: cargo-cp-artifact reads the name of crate for artifacts, so this stays
-# the same as the one specified in `../package.json`.
-name = "client"
+name = "iota-client-nodejs"
```

Comment:
should we do 
```
[package.metadata.maturin]
name = "iota-client"
```

here as well?

Reply:
Maturin is the release manager of Python, so no